packages/fuel-gauge/src/utils.ts

```diff
@@ -36,6 +37,7 @@ export type SetupConfig = {
   contractBytecode: BytesLike;
   abi: JsonAbi | Interface;
   cache?: boolean;
+  configurableConstants?: { [name: string]: unknown };
```

Comment:
Maybe I'm missing something, but this `configurableConstants` addition seems unnecessary here. 🤔

Reply:
IMO it felt like something that was missed from the `setup` function but I ended up not even using it. Will remove.

Reply:
https://github.com/FuelLabs/fuels-ts/pull/2040/commits/169489dcc1837d3df8b48d75fb7af282d8b67a8b